db/migrate/20210201161046_add_collect_signatures_to_petitions.rb

```diff
@@ -0,0 +1,5 @@
+class AddCollectSignaturesToPetitions < ActiveRecord::Migration[6.1]
+  def change
+    add_column :petitions, :collect_signatures, :boolean, default: false
```

Comment:
You probably want `null: false` there too

FYI, for future reference you'd not set the default on a column like this when deploying to a live site as it will lock the whole table. You'd write it like this:

```
# define model inline with the migration
class Petition < ActiveRecord::Base; end

def change
  add_column :petitions, :collect_signatures, :boolean

  up_only do
    Petition.find_each do |petition|
      petition.update_column(:collect_signatures, false)
    end
  end

  change_column_default :petitions, :collect_signatures, from: nil, to: false
  change_column_null :petitions, :collect_signatures, false
end
```